src/main/java/ch/njol/skript/conditions/CondIsActivated.java

```diff
@@ -0,0 +1,72 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.CreatureSpawner;
+import org.bukkit.block.data.type.TrialSpawner;
+import org.bukkit.block.data.type.TrialSpawner.State;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Is Activated")
+@Description({
+	"This condition checks whether one or more blocks are activated.",
+	"Only supports Spawners and Trial Spawners (if a player is close enough)."
+})
+@Examples({
+	"every 15 seconds:",
+		"\tif {spawners::*} are activated:",
+			"\t\tbroadcast \"All spawners are active!\""
+})
+@Since("INSERT VERSION")
+public class CondIsActivated extends Condition {
+
+	private static final boolean TRIAL_SPAWNER_EXISTS = Skript.classExists("org.bukkit.block.data.type.TrialSpawner");
+
+	static {
+		Skript.registerCondition(CondIsActivated.class, "%blocks% (is|are) activated", "%blocks% (isn't|is not|aren't|are not) activated");
```

Comment:
I think `is active` might be better than `is activated`, or both could be options. Also this could be a PropertyCondition.